.github/workflows/deploy.yml

```diff
@@ -1,91 +1,88 @@
-on: push
+name: Wrangler Action Self Testing
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
 
 jobs:
-  lint:
+  wrangler_action_self_testing:
     runs-on: ubuntu-latest
-    name: Lint
     steps:
-    - uses: actions/checkout@v2
-    - name: Lint shell script
-      uses: azohra/shell-linter@v0.3.0
-      with:
-        path: "entrypoint.sh"
-  build-only:
-    runs-on: ubuntu-latest
-    name: Only build the app
-    steps:
-      - uses: actions/checkout@v2
-      - name: Build app
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: "latest"
+          cache: "npm"
+
+      - name: Install Dependencies
+        run: npm install
+
+      - name: Build Action
+        run: npm run build
+
+      - name: Only build app
         uses: ./
         with:
+          workingDirectory: "./test/base"
           apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
-          workingDirectory: "test"
-          publish: false
-          secrets: |
-            SECRET1
-            SECRET2
-          preCommands: echo "*** pre commands ***"
+          accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
+          postCommands: npx wrangler deploy --dry-run && exit 0
```

Comment:
Doesn't `postCommands` run after `commands`? Is this under the hood doing `wrangler deploy` and then `npx wrangler deploy --dry-run`?

Reply:
Yeah that would be the idea. Can switch it to precommand which is more likely what I meant to do lol